src/test/resources/localhost_stubs/localhost_REST_fake_end_points_stubs.json

```diff
@@ -1,226 +1,255 @@
-                            "zip": "560 001"
-                        }
-                    ]
-                }
+          ]
+        }
+      }
+    },
+    {
+      "name": "bare_string_get",
+      "operation": "GET",
+      "url": "/api/v1/google-uk/employees/101",
+      "response": {
+        "status": 200,
+        "body": "I am emp No 101. Nothing more than this I can respond"
+      }
+    },
+    {
+      "name": "Sample_Get_Full_Employee_by_Id",
+      "operation": "GET",
+      "url": "/api/v1/employees/emp1001",
+      "response": {
+        "status": 200,
+        "body": {
+          "id": "emp1001",
+          "name": "Jeff Bezos",
+          "achievements": "Amazon",
+          "empType": "Self Employed",
+          "addresses": [
+            {
+              "type": "Office",
+              "line1": "39 NewYork Street",
+              "zip": "560 390"
+            },
+            {
+              "type": "Home",
+              "line1": "99 Piscataway",
+              "zip": "560 001"
             }
-        },
-        {
-            "name": "Mock GET employee details including DOB",
-            "operation": "GET",
-            "url": "/api/v1/google-uk/employees/UK-LON-1002",
-            "response": {
-                "status": 200,
-                "body": {
-                    "empId": "UK-LON-1002",
-                    "city": "UK-London",
-                    "dob": "1989-07-09"
-                }
+          ]
+        }
+      }
+    },
+    {
+      "name": "Sample_Get_Address_by_emp_id",
+      "operation": "GET",
+      "url": "/api/v1/addresses/empoyee/emp1001",
+      "response": {
+        "status": 200,
+        "body": {
+          "empId": "emp1001",
+          "addresses": [
+            {
+              "addressId": "addr-001",
+              "type": "Office",
+              "line1": "39 NewYork Street",
+              "zip": "560 390"
+            },
+            {
+              "addressId": "addr-002",
+              "type": "Home",
+              "line1": "99 Piscataway",
+              "zip": "560 001"
             }
+          ]
+        }
+      }
+    },
+    {
+      "name": "Mock GET employee details including DOB",
+      "operation": "GET",
+      "url": "/api/v1/google-uk/employees/UK-LON-1002",
+      "response": {
+        "status": 200,
+        "body": {
+          "empId": "UK-LON-1002",
+          "city": "UK-London",
+          "dob": "1989-07-09"
+        }
+      }
+    },
+    {
+      "name": "Mock POST to OAuth2 server",
+      "operation": "POST",
+      "url": "/oauth/v2/token?refresh_token=refresh.token1224454564657556&client_id=client.id74528572945820&client_secret=client.secrect14879452304958245&grant_type=refresh_token",
+      "response": {
+        "status": 200,
+        "body": {
+          "access_token": "1000.59d663ba09e17428a405c37435k3uhf58affd.554a1cefccd19fdgi7ef1db62f71d4393e",
+          "expires_in_sec": 3600,
```

Comment:
Is this value somewhere used to refresh the token? Just checking, in live system whether this will be used.
- expires_in_sec
- expires_in

Reply:
yes, expires_in should be used. But for simplicity, I have put it as constant.